CONP_DATS_fields.md

```diff
@@ -36,10 +35,7 @@ The information will be used to create your resource landing page on the CONP po
 |aggregation|**OPTIONAL**. Qualifier indicating whether the entity represents an 'instance of a dataset' or a 'collection of datasets'.|
 |spatialCoverage|**OPTIONAL**. The geographical extension and span covered by the dataset and its measured dimensions/variables.|
 
-<!--- how to acknowledge
-RECOMMENDED.  Instructions how researchers using this dataset should acknowledge the original authors. This field can also be used to define a publication that should be cited in publications that use the dataset. --->
-<!--- availability
-OPTIONAL. A qualifier indicating the different types of availability for a dataset (available, unavailable, embargoed, available with restriction, information not available) --->
+The DATS dataset schema can be found [here](https://github.com/CONP-PCNO/schema/blob/master/dataset_schema.json), and the `DATS.json` file from the visual-working-memory dataset [here](https://github.com/emmetaobrien/ds001634/blob/4f9721a053a3c211b45c6b3f61e8b2cc69027c74/DATS.json) can be used as a template. A graphic interface allowing users to fill in fields online is under development [here](https://dats-creator.herokuapp.com/).
```

Comment:
is there a reason why the link for the visual-working-memory dataset points to your repo instead of a conp-dataset?

Reply:
I had not got around to forking that recent regeneration of visual-working-memory into conpdatasets yet, I will do that now and then edit the link.

Reply:
Can you merge https://github.com/conpdatasets/visual-working-memory/pull/6/ ?